### PR TITLE
Create plot visualization 38

### LINKED ISSUE #38: Create plot visualizations
Add simple plot visuals to the site. Write components in a way so that they're well contained, so that we can swap out the plotting library in the future as needed, as requirements may change.

Would recommend using [plotly](https://www.npmjs.com/package/plotly.js/v/1.47.4) as a starting point as it's a nice high level library that has the functionality that we need, but am open to other suggestions.

### PR DESCRIPTION
**Pull request by bishwobhandari**
closes #38 
plot visualization plotly js added through cdn.

### CODE REVIEW

File: frontend/src/stores/OutputStore.js [hunk 1]
```diff
@@ -228,6 +247,101 @@ class OutputStore {
...
+        },
+    };
+
+    @action addPlotData = index => {
+        let output = this.getCurrentOutput(this.selectedDatasetIndex);
+        let currentModel = output.models.find(item => item.model_index == index);
+        let cdf = currentModel.results.cdf;
+        let doses = output.dataset.doses;
+
+        let doseRange = _.max(doses) - _.min(doses);
```
Comment (bishwobhandari) [inline frontend/src/stores/OutputStore.js:319]: for best fit curve

File: frontend/src/stores/OutputStore.js [hunk 2]
```diff
@@ -228,6 +247,101 @@ class OutputStore {
...
+            xArray.push(new_item);
+        });
+
+        var trace2 = {
+            x: xArray,
+            y: cdf,
+            mode: "marker",
+            type: "line",
+        };
+
```
Comment (bishwobhandari) [inline frontend/src/stores/OutputStore.js:333]: review for fitting curve

File: frontend/src/stores/OutputStore.js [hunk 3]
```diff
@@ -228,6 +247,101 @@ class OutputStore {
...
+        let output = this.getCurrentOutput(this.selectedDatasetIndex);
+        let currentModel = output.models.find(item => item.model_index == index);
+        let cdf = currentModel.results.cdf;
+        let doses = output.dataset.doses;
+
+        let doseRange = _.max(doses) - _.min(doses);
+        let pValue = this.getPValue;
+        let xArray = [];
+        pValue.map(item => {
+            let new_item = (item / 100) * doseRange;
```
Comment (shapiromatron) [inline frontend/src/stores/OutputStore.js:323]: so this should be the array of cdf, not pvalues. 

- for x, you use the index of the value (so b/w 0-99 and multiply by doseRange)
- for y, you'll calculate a responseRange just like dose-range that uses either response mean for continuous data or incidence/n for dichotomous data, and you'll multiple it by the value

**Comment by shapiromatron** [comment]
@bishwobhandari i think this branch has some issues w/o old changes, can those be removed pls?

File: bmds_server/jobrunner/views.py [hunk 4]
```diff
@@ -40,6 +40,7 @@ def get_redirect_url(self, *args, **kwargs):
 
 class JobDetail(DetailView):
     model = models.Job
+    print("job detail class")
```
Comment (shapiromatron) [inline bmds_server/jobrunner/views.py:43]: remove

Comment (shapiromatron) [reply bmds_server/jobrunner/views.py:43]: it might just be this change. there's some other css and style changes, are those intentional?

Comment (bishwobhandari) [reply bmds_server/jobrunner/views.py]: removed. some of the css was updated.

File: bmds_server/static/css/style.css [hunk 5]
```diff
@@ -0,0 +1,542 @@
+body{
```
Comment (shapiromatron) [inline bmds_server/static/css/style.css:1]: delete? this isn't used anywhere?

**Pull request merged**